app/src/main/java/com/cmput301w23t09/qrhunter/qrcode/AddQRCodeFragment.java

```diff
@@ -106,16 +111,6 @@ private void onAddQRClicked(View view) {
                             loadingButton.setVisibility(View.GONE);
                             return;
                           }
-
-                          // Add the player to the QR
-                          QRCodeDatabase.getInstance()
-                              .addPlayerToQR(
-                                  activePlayer,
-                                  qrCode,
-                                  ignored -> {
-                                    loadingButton.setVisibility(View.GONE);
-                                    this.dismiss();
-                                  });
```

Comment:
Why was this removed?

Reply:
My bad there, was merging on her branch and I had thought it was useless during my merging. will fix.